bundle/manifests/cache.open-cluster-management.io_ocmsearches.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: ocmsearches.cache.open-cluster-management.io
+spec:
+  group: cache.open-cluster-management.io
+  names:
+    kind: OCMSearch
```

Comment:
Can we rename this crd to `Search` ?  The OCM qualifier is defined by the group, so OCMSearch is redundant.

Reply:
We have not completely defined the CRD . We can rename as Search